gson/testcases-proguard.conf

```diff
@@ -0,0 +1,20 @@
+# Options from Android Gradle Plugins
```

Comment:
Would it be possible to move this to `gson/src/test/resources`? I think that might be better than having it directly in the `gson` folder.

Reply:
Fixed at a25f96b8936625ff4856495e8f8dfc63205bcbd7